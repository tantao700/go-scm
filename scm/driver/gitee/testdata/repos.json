[{
    "id": 1296269,
    "owner": {
        "login": "octocat",
        "id": 1,
        "avatar_url": "https://github.com/images/error/octocat_happy.gif",
        "gravatar_id": "",
        "url": "https://gitee.com/api/v5/users/octocat",
        "html_url": "https://github.com/octocat",
        "followers_url": "https://gitee.com/api/v5/users/octocat/followers",
        "following_url": "https://gitee.com/api/v5/users/octocat/following{/other_user}",
        "gists_url": "https://gitee.com/api/v5/users/octocat/gists{/gist_id}",
        "starred_url": "https://gitee.com/api/v5/users/octocat/starred{/owner}{/repo}",
        "subscriptions_url": "https://gitee.com/api/v5/users/octocat/subscriptions",
        "organizations_url": "https://gitee.com/api/v5/users/octocat/orgs",
        "repos_url": "https://gitee.com/api/v5/users/octocat/repos",
        "events_url": "https://gitee.com/api/v5/users/octocat/events{/privacy}",
        "received_events_url": "https://gitee.com/api/v5/users/octocat/received_events",
        "type": "User",
        "site_admin": false
    },
    "name": "Hello-World",
    "full_name": "tantao700/hello-world",
    "description": "This your first repo!",
    "private": true,
    "fork": true,
    "url": "https://gitee.com/api/v5/repos/tantao700/hello-world",
    "html_url": "https://github.com/tantao700/hello-world",
    "archive_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/{archive_format}{/ref}",
    "assignees_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/assignees{/user}",
    "blobs_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/git/blobs{/sha}",
    "branches_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/branches{/branch}",
    "clone_url": "https://github.com/tantao700/hello-world.git",
    "collaborators_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/collaborators{/collaborator}",
    "comments_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/comments{/number}",
    "commits_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/commits{/sha}",
    "compare_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/compare/{base}...{head}",
    "contents_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/contents/{+path}",
    "contributors_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/contributors",
    "deployments_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/deployments",
    "downloads_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/downloads",
    "events_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/events",
    "forks_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/forks",
    "git_commits_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/git/commits{/sha}",
    "git_refs_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/git/refs{/sha}",
    "git_tags_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/git/tags{/sha}",
    "git_url": "git:github.com/tantao700/hello-world.git",
    "hooks_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/hooks",
    "issue_comment_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/issues/comments{/number}",
    "issue_events_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/issues/events{/number}",
    "issues_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/issues{/number}",
    "keys_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/keys{/key_id}",
    "labels_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/labels{/name}",
    "languages_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/languages",
    "merges_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/merges",
    "milestones_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/milestones{/number}",
    "mirror_url": "git:git.example.com/tantao700/hello-world",
    "notifications_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/notifications{?since, all, participating}",
    "pulls_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/pulls{/number}",
    "releases_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/releases{/id}",
    "ssh_url": "git@github.com:tantao700/hello-world.git",
    "stargazers_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/stargazers",
    "statuses_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/statuses/{sha}",
    "subscribers_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/subscribers",
    "subscription_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/subscription",
    "svn_url": "https://svn.github.com/tantao700/hello-world",
    "tags_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/tags",
    "teams_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/teams",
    "trees_url": "http://gitee.com/api/v5/repos/tantao700/hello-world/git/trees{/sha}",
    "homepage": "https://github.com",
    "language": null,
    "forks_count": 9,
    "stargazers_count": 80,
    "watchers_count": 80,
    "size": 108,
    "default_branch": "master",
    "open_issues_count": 0,
    "topics": [
        "octocat",
        "atom",
        "electron",
        "API"
    ],
    "has_issues": true,
    "has_wiki": true,
    "has_pages": false,
    "has_downloads": true,
    "archived": false,
    "pushed_at": "2011-01-26T19:06:43Z",
    "created_at": "2011-01-26T19:01:12Z",
    "updated_at": "2011-01-26T19:14:43Z",
    "permissions": {
        "admin": true,
        "push": true,
        "pull": true
    },
    "allow_rebase_merge": true,
    "allow_squash_merge": true,
    "allow_merge_commit": true,
    "subscribers_count": 42,
    "network_count": 0,
    "license": {
        "key": "mit",
        "name": "MIT License",
        "spdx_id": "MIT",
        "url": "https://gitee.com/api/v5/licenses/mit",
        "html_url": "http://choosealicense.com/licenses/mit/"
    }
}]