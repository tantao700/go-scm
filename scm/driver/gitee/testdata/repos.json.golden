[
    {
        "ID": "178504",
        "Namespace": "diaspora",
        "Name": "diaspora",
        "Perm": {
            "Pull": true,
            "Push": false,
            "Admin": false
        },
        "Branch": "master",
        "Private": false,
        "Clone": "https://gitee.com/mirrors/diaspora.git",
        "CloneSSH": "git@gitee.com:mirrors/diaspora.git",
        "Link": "https://gitee.com/mirrors/diaspora",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    }
]